CONTRIBUTING.md

```diff
@@ -65,6 +65,10 @@ Other Tips
 
         $ git commit --amend
 
+  - Then force push your branch:
```

Comment:
Of course, if they haven't yet pushed to a remote there's no reason to use `--force` in the first place unless they also rebased for some other reason.  So, maybe the whole thing is moot...